polkadot-parachain/src/command.rs

```diff
@@ -56,6 +56,7 @@ enum Runtime {
 	ContractsRococo,
 	CollectivesPolkadot,
 	CollectivesWestend,
+	BridgeHub(chain_spec::bridge_hubs::BridgeHubRuntimeType),
```

Comment:
Can we rename `BridgeHubRuntimeType` to be `RelayType` as I can imagine that we will reuse this?

Reply:
good point, I never realized it copies the relay type :D, I will check this later